cf/manifest/manifest_test.go

```diff
@@ -248,8 +248,8 @@ var _ = Describe("Manifests", func() {
 
 		Expect(*apps[0].BuildpackUrl).To(Equal("my-buildpack"))
```

Comment:
Maybe it's just me but I kind of find this `*apps[0]` business ugly...  especially since it's repeated. Maybe have a local var `apps0 := *apps[0]` or similar.

No big deal, just my .02